packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -265,17 +318,17 @@ describe('Loan', () => {
     expect(typeof txId).toStrictEqual('string')
     expect(txId.length).toStrictEqual(64)
     await tGroup.get(0).generate(1)
-  })
+  }, 580000)
```

Comment:
Hmm, ran 3 times without the `580000` timeout given. tests executed and passed fine with ~`158s`.🤔